spec/index.html

```diff
@@ -612,6 +624,49 @@ <h3>Version Labels</h3>
         </tr>
       </tbody>
     </table>
+
+    <p>For serializations supporting in-line version annouoncement,
+      the version announcement SHOULD be made early in the document
+      and certainly before serializing any feature depending on that version.</p>
+
+    <section id="server-considerations" class="informative">
+      <h4>Server considerations</h4>
```

Comment:
To be consistent with the capitalization of the other section titles in the document:
```suggestion
      <h4>Server Considerations</h4>
```